src/base/utils/misc.cpp

```diff
@@ -503,9 +503,9 @@ QList<bool> Utils::Misc::boolListfromStringList(const QStringList &l)
 
 bool Utils::Misc::isUrl(const QString &s)
 {
-    const QString scheme = QUrl(s).scheme();
-    QRegExp is_url("http[s]?|ftp", Qt::CaseInsensitive);
-    return is_url.exactMatch(scheme);
+    static const QRegExp isURL("http[s]?|ftp", Qt::CaseInsensitive);
```

Comment:
>I don't see any reason why it shouldn't be static. What do you think?

I'll let you decide, since I can't argue the slowness of `QRegExp` is at construction or `QRegExp.exactMatch()` (hence the suggestion above).